library/Icingadb/Widget/Detail/UserDetail.php

```diff
@@ -0,0 +1,110 @@
+    {
+        $this->user = $user;
+    }
+
+    protected function createCustomVars()
+    {
+        $content = [new HtmlElement('h2', null, Text::create(t('Custom Variables')))];
+        $flattenedVars = $this->user->customvar_flat;
+        $this->applyRestrictions($flattenedVars);
+
+        $vars = $this->user->customvar_flat->getModel()->unflattenVars($flattenedVars);
+        if (! empty($vars)) {
+            $customvarTable = new CustomVarTable($vars);
+            $customvarTable->setAttribute('id', 'user-customvars');
+            $content[] = $customvarTable;
+        } else {
+            $content[] = new EmptyState(t('No custom variables configured.'));
+        }
+
+        return $content;
+    }
+
+    protected function createUserDetail()
+    {
+        return [
+            (new HtmlElement('h2', null, Text::create(t('Details')))),
+            (new HtmlElement(
+                'ul',
+                Attributes::create(['class' => 'key-value-list']),
+                (new HtmlElement(
+                    'li',
+                    null,
+                    (new HtmlElement(
+                        'span',
+                        Attributes::create(['class' => 'label']),
+                        Text::create(t('E-Mail'))
+                    )),
+                    (new HtmlElement(
+                        'span',
+                        Attributes::create(['class' => 'value']),
+                        $this->user->email
+                            ? Text::create(t($this->user->email))
+                            : (new HtmlElement(
+                                'span',
+                                Attributes::create(['class' => 'text-muted']),
+                                Text::create(t('Unset'))
+                            ))
+                    ))
+                )),
+                (new HtmlElement(
+                    'li',
+                    null,
+                    (new HtmlElement(
+                        'span',
+                        Attributes::create(['class' => 'label']),
+                        Text::create(t('Pager'))
+                    )),
+                    (new HtmlElement(
+                        'span',
+                        Attributes::create(['class' => 'value']),
+                        $this->user->pager
+                            ? Text::create(t($this->user->pager))
+                            : (new HtmlElement(
+                                'span',
+                                Attributes::create(['class' => 'text-muted']),
+                                Text::create(t('Unset'))
+                            ))
+                    ))
+                ))
+            ))
+        ];
+    }
+
+    protected function assemble()
+    {
+        $this->add([
+            $this->createUserDetail(),
+            (new HtmlElement('h2', null, Text::create(t('User Groups')))),
+            new UsergroupList($this->user->usergroup),
```

Comment:
This is more of a `createUsergroupList()`. And while you're at it, also add the `ShowMore` widget to it.